### ISSUE TITLE
Touch target for checkbox in tree view doesn't include text label

### ISSUE THREAD

**Issue by dexfab** [vmware/clarity#4067]
## Describe the bug

Checkboxes in a tree view are not consistent with standard checkboxes as the touch target does not include the text label.

## How to reproduce

1.  Go to https://v2.clarity.design/tree-view#checkbox-tree
2.  Click on chekbox
3.  Hover on associated text label
4.  Label is not clickable

## Expected behavior

As per standard checboxes https://v2.clarity.design/checkboxes, label should be clickable.

## Versions

* Clarity: 2.3.2

**Device:**

* Browser: Chrome

**Comment by hippee-lee** [comment]
It looks like tree-node needs to be re-factored to use labels inside of a clr-checkbox-wrapper container.

**Comment by gnomeontherun** [comment]
Clarity cares deeply about accessibility, and we work with our accessibility team to identify and triage issues. We have an internal tracking system to identify and verify accessibility issues, which has captured this already. Some issues with Clarity Angular are difficult to solve without causing side effects or breaking changes, and we have to balance concerns with fixing issues and breaking changes. As we build out Clarity Core, we are able to address these root cause accessibility issues and recommend adopting the Clarity Core implementations as they become available. We are closing this issue as we have the issues tracked internally. Please follow our development and releases to see when we release relevant Clarity Core components to make this possible.
